packages/ecc-client-ga4gh-wes/src/components/wes-run/definition/wesRun.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  FASTElement,
+  customElement,
+  attr,
+  observable,
+} from '@microsoft/fast-element';
+import template from './wesRun.template.js';
+import styles from './wesRun.styles.js';
+import WorkflowData from './workflowData.js';
+import { cancelWorkflow, fetchWorkflow } from '../../../data/index.js';
+
+@customElement({
+  name: 'ecc-client-ga4gh-wes-run',
+  template,
+  styles,
+})
+export default class WESRun extends FASTElement {
+  @attr baseURL = '';
+
+  @attr id = '';
+
+  @attr state = '';
+
+  @observable isLoading = true;
+
+  @observable data: WorkflowData = {
+    run_id: this.id,
+    state: this.state,
+    request: {
+      workflow_params: {},
+      workflow_type: '',
+      workflow_type_version: '',
+      tags: {
+        additionalProp1: '',
+        additionalProp2: '',
+        additionalProp3: '',
+      },
+      workflow_engine_parameters: {
+        additionalProp1: '',
+        additionalProp2: '',
+        additionalProp3: '',
+      },
+      workflow_url: '',
+    },
+    run_log: {
+      name: '',
+      cmd: [],
+      start_time: '',
+      end_time: '',
+      stdout: '',
+      stderr: '',
+      exit_code: 0,
+    },
+    task_logs: [
+      {
+        name: '',
+        cmd: [],
+        start_time: '',
+        end_time: '',
+        stdout: '',
+        stderr: '',
+        exit_code: 0,
+      },
+    ],
+    outputs: {},
+  };
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.handleFetch();
+  }
+
+  disconnectedCallback() {
```

Comment:
are you using this for anything?

Reply:
PR wasn't yet ready for review; this is just the setup, not the component.